File: UMRPublicSPA/app/jobs-opening/model/job-content.ts
export class JobContent {
    public JobContentId: number;
    public JobTitle: string;
    public NoOfVacancies: string;
    public JobDescription: string;
    public JobNature: string;
    public EducationalRequirements: string;
    public ExperienceRequirements: string;
    public AdditionalJobRequirements: string;
    public JobLocation: string;
    public SalaryRange: string;
    public OtherBenefits: string;
    public EmailCV: string;
    public SpecialInstruction: string;
    public ApplicationDeadline: string;
    public HalfContent: string;
    public SecondHalf: string;

    //get halfContent(): string {
    //    var len = this.JobDescription.length;
    //    console.log('From Object: ' + this._halfContent.substring(0, len / 2));
    //    return this._halfContent.substring(0, len / 2);
    //}
    //set halfContent(halfContent: string) {
    //    this._halfContent = halfContent;
    //}        
}

export interface ColumnOptions {
    header: string,
    field: string,
    sortable: boolean
};

export interface GridColumns {
    JobContentId: number,
    JobTitle: string
};

export interface GridConfiguration {
    page: number,
    pageSize: number,
    sort: string,
    order: number
};

File: JobAPI/Models/JobContent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PublicJobAPI.Models
{
    public class JobContent
    {
        public int JobContentId { get; set; }
        public string JobTitle { get; set; }
        public string NoOfVacancies { get; set; }
        public string JobDescription { get; set; }
        public string JobNature { get; set; }
        public string EducationalRequirements { get; set; }
        public string ExperienceRequirements { get; set; }
        public string AdditionalJobRequirements { get; set; }
        public string JobLocation { get; set; }
        public string SalaryRange { get; set; }
        public string OtherBenefits { get; set; }
        public string EmailCV { get; set; }
        public string SpecialInstruction { get; set; }
        public string ApplicationDeadline { get; set; }
        public string CreatedDate { get; set; }
        public string County { get; set; }
        public string StateCode { get; set; }
        public string StateName { get; set; }
    }
}